docs/getting-started.md

```diff
@@ -8,8 +8,19 @@ Extensions are stored in `.vscode/extenstions.json`, and will automatically be s
 
 ## DataHub Application (app-dh)
 
-Deployed version of the application can be found [here.](https://cdn-dh-endpoint-dh-u-001.azureedge.net/)  
-Currently the application is only deployed to a single environment.
+Deployed version of the application can be found here:
+
+- [u001](https://jolly-sand-03f839703.azurestaticapps.net)
+- [u002](https://ambitious-coast-027d0aa03.azurestaticapps.net)
+- [t001](https://lively-river-0f22ad403.azurestaticapps.net)
+- [b001](https://blue-rock-05b7e5e03.azurestaticapps.net)
+- [b002](https://purple-forest-07e41fb03.azurestaticapps.net)
```

Comment:
nitpick: capitalize the letters in the beginning. Also in the note below. Otherwise looks good.

unrelated: Really like the naming - "jolly-sand", "ambitious-coast", ... 😂